docs/tests/manual-scenarios.md

```diff
@@ -0,0 +1,93 @@
+**Nazwa scenariusza**: Logowanie
+
+**ID**: S1
+
+**Opis**: Zweryfikowanie poprawności działania procesu logowania
+
+**Czynności przygotowawcze**: Utworzenie konta o danych `person/apsi`
+
+**Czynności końcowe**: Usunięcie konta `person/apsi` (o ile wcześniej nie istniało)
+
+**Przypadki testowe scenariusza**:
+
+| ID i Nazwa Przypadku Testowego           | Warunki wstępne                    | Dane testowe                              | Kroki reprodukcji                                              | Oczekiwany rezultat                                               |
+|------------------------------------------|------------------------------------|-------------------------------------------|----------------------------------------------------------------|-------------------------------------------------------------------|
+| TC1.1 Poprawne logowanie                 | 1. Potrzebne jest istniejące konto | 1. LOGIN: "person"<br>2. HASŁO: "apsi"    | 1. Wprowadź login<br>2. Wprowadź hasło<br>3. Kliknij "Zaloguj" | 1. Pomyślny login                                                    |
+| TC1.2 Logowanie do nieistniejącego konta |                                    | 1. LOGIN: "wrong"<br>2. HASŁO: "apsi"     | 1. Wprowadź login<br>2. Wprowadź hasło<br>3. Kliknij "Zaloguj" | 1. Pokazanie komunikatu "Niepoprawny login lub hasło"                |
+| TC1.3 Logowanie do konta ze złym hasłem  | 1. Potrzebne jest istniejące konto | 1. LOGIN: "person"<br>2. HASŁO: "apsi"    | 1. Wprowadź login<br>2. Wprowadź hasło<br>3. Kliknij "Zaloguj" | 1. Pokazanie komunikatu "Niepoprawny login lub hasło"                |
+| TC1.4 Login z pustymi danymi             |                                    | 1. LOGIN: "doesnot"<br>2. HASŁO: "matter" | 1. Wprowadź login<br>2. Kliknij "Zaloguj"<br>3. Usuń login     | 1. Pokazanie komunikatów "Należy podać login" i "Należy podać hasło" |
```

Comment:
jak nie wprowadzamy jakiejś wartości to myślę że bez sensu ją podawać

Reply:
Prawda, poprawione